### DISCUSSION TITLE
[FEATURE REQUEST] Option to suppress audio downloads

### DISCUSSION THREAD

**Discussion by FLPresets** [mastodon/mastodon#21485] [General]
Is it possible to add an option to suppress audio downloads? 

Musicians would feel more comfortable putting music/media on Mastodon if they didn't need to worry as much about piracy/people downloading their music when they don't want it available to be downloaded.

Right now, there's a blatant "download" option on every audio upload.

**Comment by Kaelph** [comment]
I second this, it would be nice to have the option to have the audio not be downloadable.
